plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/StaticTokenAwareMetastoreClientFactory.java

```diff
@@ -144,9 +143,9 @@ private static URI checkMetastoreUri(URI uri)
         requireNonNull(uri, "uri is null");
         String scheme = uri.getScheme();
         checkArgument(!isNullOrEmpty(scheme), "metastoreUri scheme is missing: %s", uri);
-        checkArgument(scheme.equals("thrift"), "metastoreUri scheme must be thrift: %s", uri);
+        checkArgument(scheme.equals("thrift") || scheme.equals("https") || scheme.equals("http"), "metastoreUri scheme must be thrift, http or https: %s", uri);
```

Comment:
Removing this check for https and http because this class only works with thrift now.